src/main/java/com/ldtteam/structurize/config/ServerConfiguration.java

```diff
@@ -59,6 +64,7 @@ protected ServerConfiguration(final ForgeConfigSpec.Builder builder)
         maxCachedSchematics = defineInteger(builder, "maxCachedSchematics", 100, 0, 100000);
         maxBlocksChecked = defineInteger(builder, "maxBlocksChecked", 1000, 0, 100000);
         schematicBlockLimit = defineInteger(builder, "schematicBlockLimit", 100000, 1000, 1000000);
+        iterationVersion = defineInteger(builder, "iterationVersion", 1, 1, 2);
```

Comment:
This isn't especially user-friendly.  Though I suppose it could be explained in comments.

Reply:
Actually, I did set that on purpose for the time being. I want this added initially that people that are open for experiments can test and not sth people in general should be playing around with on/off.